### PR TITLE
Make get_latest_gv_version check for both geckoview and geckoview-omn…

### PR DESCRIPTION
**Pull request by jcristau**
…i (bug 1780820)

A-C builds need geckoview-omni, which depends on geckoview-exoplayer2.
The latter is built/uploaded with the gv lite runtime and so can be out
of sync with the "omni" build.

### CODE REVIEW

File: src/util.py [hunk 1]
```diff
@@ -146,18 +146,24 @@ def get_latest_gv_version(gv_major_version, channel):
...
 
     r = requests.get(
         f"{MAVEN}/org/mozilla/geckoview/{name}/maven-metadata.xml"
     )
     r.raise_for_status()
     metadata = xmltodict.parse(r.text)
+    r = requests.get(
+        f"{MAVEN}/org/mozilla/geckoview/{name_lite}/maven-metadata.xml"
+    )
+    r.raise_for_status()
```
Comment (JohanLorenzo) [inline src/util.py:161]: Improvement for later (not in this PR): retry failed requests to be more resilient to network errors.

File: src/util.py [hunk 2]
```diff
@@ -146,18 +146,24 @@ def get_latest_gv_version(gv_major_version, channel):
...
+    lite_metadata = xmltodict.parse(r.text)
 
-    versions = []
-    for version in metadata["metadata"]["versioning"]["versions"]["version"]:
-        if version.startswith(f"{gv_major_version}."):
-            versions.append(version)
+    versions = [v for v in metadata["metadata"]["versioning"]["versions"]["version"]
+                if v.startswith(f"{gv_major_version}.")
+                and v in lite_metadata["metadata"]["versioning"]["versions"]["version"]]
```
Comment (JohanLorenzo) [inline src/util.py:166]: Good call on making sure the exact version exist in both metadata sets.

File: src/util.py [hunk 3]
```diff
@@ -146,18 +146,24 @@ def get_latest_gv_version(gv_major_version, channel):
         name += "-" + channel
     # A-C builds against geckoview-omni
     # See https://github.com/mozilla-mobile/android-components/commit/0b349f48c91a50bb7b4ffbf40c6c122ed18142d3
+    # However, geckoview-omni requires exoplayer2 which comes from the lite build, so check for that too
+    name_lite = name
     name += "-omni"
```
Comment (JohanLorenzo) [inline src/util.py:151]: Note to self: I didn't remember what Python does with strings and the `+=` operator. I tested`name_lite` keeps the old value.

**Review by JohanLorenzo — APPROVED** [review]
🚢 it!

File: src/util.py (hunk 1 above)
Comment (jcristau) [reply src/util.py:161]: The action runs every 15 minutes, so I don't think we need additional retries.

**Pull request merged**